app/models/orm/migrations/env.py

```diff
@@ -98,7 +98,7 @@ def run_migrations_online():
         with context.begin_transaction() as transaction:
             context.run_migrations()
             if "dry-run" in context.get_x_argument():
-                print("Dry-run succeeded; now rolling back transaction")
+                print("Dry-run succeeded; now rolling back transactionapp")
```

Comment:
i know this isn't part of your PR changes, @dmannarino, but is this a typo?

Reply:
It is. I didn't bother to fix it as I will be reverting the commit anyway.